fixtures/external-dispatch-namespace-app/tests/index.test.ts

```diff
@@ -98,7 +107,7 @@ describe("external-dispatch-namespace-app", () => {
 			`"p2"`
 		);
 		expect(await dispatcherResponse.text()).toMatchInlineSnapshot(
-			`"intercepted"`
+			`"{\"accept\":\"*/*\",\"accept-encoding\":\"br, gzip\",\"accept-language\":\"*\",\"host\":\"localhost:${outbound.port}\",\"sec-fetch-mode\":\"cors\",\"user-agent\":\"undici\",\"x-foo\":\"bar\"}"`
```

Comment:
I'm not sure if `host` here should be preserved or overwritten. Need to check what prod runtime behavior is.